<launch>
  <arg name="camera" default="camera"/>
  <arg name="image_topic" default="image_rect"/>
  <arg name="depth_topic" default="image_rect"/>
  <!--include file="$(find openni_launch)/launch/openni.launch"/-->
  <param name="/camera/depth_registration" type="bool" value="true" />

  <node pkg="face_detector" type="face_detector" name="face_detector" launch-prefix="nice">
     <remap from="camera" to="$(arg camera)"/>
     <remap from="image" to="$(arg image_topic)"/>
     <remap from="depth" to="$(arg depth_topic)"/>
  	 <param name="classifier_name" type="string" value="frontalface" />
	 <param name="classifier_filename" type="string" value="$(env ROS_ROOT)/../OpenCV/haarcascades/haarcascade_frontalface_alt.xml" />
<!-- 	 <param name="classifier_filename" type="string" value="$(find face_detector)/haarcascade_frontalface_alt.xml" />
 -->	 <param name="classifier_reliability" type="double" value="0.9"/>
	 <param name="do_continuous" type="bool" value="true" />
  	 <param name="do_publish_faces_of_unknown_size" type="bool" value="false" />	
	 <param name="do_display" type="bool" value="true" />
	 <param name="use_rgbd" type="bool" value="true" />
	 <param name="approximate_sync" type="bool" value="true" />
  </node>
</launch>
